source/configure/environment-configuration-settings.rst

```diff
@@ -4134,6 +4134,58 @@ Allow untrusted internal connections
 |   but not ``10.0.16.20``.                                                                                                                     |
 +-----------------------------------------------+-----------------------------------------------------------------------------------------------+
 
+Mobile Security
+---------------
+
+.. config:setting:: mobile-security-enabled
+  :displayname: Enable Biometric Authentication
+  :systemconsole: Environment > Mobile Security
+  :configjson: .NativeAppSettings.MobileEnableBiometrics
+  :environment: MM_NATIVEAPPSETTINGS_MOBILEENABLEBIOMETRICS
+  :description: Enforces biometric authentication (with PIN/passcode fallback) before accessing the app. Users will be prompted based on session activity and server switching rules.
```

Comment:
not sure if is important to be more specific as to when it will prompt ?

Reply:
When will they prompt?

Reply:
@cwarnermm details in this ticket https://mattermost.atlassian.net/browse/MM-63362